cluster/state_test.go

```diff
@@ -0,0 +1,32 @@
+package cluster
+
+import (
+	"net"
+	"reflect"
+	"testing"
+
+	"github.com/costela/wesher/common"
+)
+
+func Test_state_save_soad(t *testing.T) {
+	key := "abcdefghijklmnopqrstuvwxyzABCDEF"
+	node := common.Node{
+		Name: "node",
+		Addr: net.ParseIP("10.0.0.2"),
+	}
+
+	cluster := Cluster{
+		state: &state{
+			ClusterKey: []byte(key),
+			Nodes:      []common.Node{node},
+		},
+	}
+
+	cluster.state.save()
```

Comment:
```suggestion
	if err := cluster.state.save(); err != nil {
		t.Error(err)
	}
```

If the error is really just a permission issue on the folder, we could just make `statePath` a variable and change it to `/tmp/wesher.state.json` or something like that during testing.

This will change after #38 anyway.